texar/torch/modules/networks/conv_networks.py

```diff
@@ -172,10 +180,29 @@ def default_hparams():
                Keyword arguments for the activation following the convolutional
                layer. See :func:`~texar.torch.core.get_layer` for more details.
 
-           `"other_conv_kwargs"`: dict, optional
+           `"other_conv_kwargs"`: list or dict, optional
                Other keyword arguments for :torch_nn:`Conv1d` constructor,
                e.g., ``padding``.
 
+               - If a dict, the same dict is applied to all the convolution
+                 layers.
+
+               - If a list, the length must equal ``"num_conv_layers"``. This
+                 list can contain nested lists. If the convolution layer at
+                 index i has multiple kernel sizes, then the corresponding
+                 element of this list can also be a list of length equal to
+                 ``"kernel_size"`` at index i. If the element at index i is
+                 instead a dict, then the same dict gets applied to all the
+                 convolution layers at index i.
+
+           `"data_format"`: str, optional
+               Data format of the input tensor. Defaults to ``channels_first``
+               denoting the first dimension to be the channel dimension. Set it
+               to ``channels_last`` to treat last dimension as the channel
+               dimension. This argument can also be passed in ``forward``
+               function, in which case the value specified here will be
+               ignored.
```

Comment:
Same here, make the expected input size clear.

Reply:
Updated in d2c2634